surface-model/surface-model_shacl.ttl

```diff
@@ -0,0 +1,141 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix surface-model: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/surface-model/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix georeference: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/> .
+
+surface-model:surfaceModelShape a sh:NodeShape ;
```

Comment:
Classnames in PascalCase
--> SurfaceModelShape

Reply:
fixed